lib/fencing.py.py

```diff
@@ -1268,7 +1268,7 @@ def fence_logout(conn, logout_string, sleep=0):
 def source_env(env_file):
     # POSIX: name shall not contain '=', value doesn't contain '\0'
     output = subprocess.check_output("source {} && env -0".format(env_file), shell=True,
-                          executable="/bin/sh")
+                          executable="/bin/bash")
```

Comment:
in that case, the bash´ism should be removed. /bin/bash is not available on all systems.

Reply:
It works fine if you replace "source" with "." (tested with bash, dash and sh on FreeBSD):
`output = subprocess.check_output(". {} && env -0".format(env_file), shell=True, executable="/bin/sh")`